riotfile.py

```diff
@@ -288,14 +288,20 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                 "gunicorn": latest,
                 "flask": "<=2.2.3",
                 "httpretty": "<1.1",
+                "werkzeug": "<2.0",
+                "pytest-randomly": latest,
```

Comment:
Can we move this dependency to the base riot venv

Reply:
Do you mean the one defined on line 91? Updating that one causes all of the envs to be randomized, which I'm avoiding by splitting that change into multiple pull requests that are easier to review.